### PR TITLE
Add functions for creating artificial typos

### PR DESCRIPTION
**Pull request by irinakhismatullina**

### CODE REVIEW

File: lookout/style/typos/corruption.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+from lookout.style.typos.utils import Columns
+
+
+letters = list(string.ascii_lowercase)
+
+
+def rand_insert(token: str) -> str:
+    """
+    Add random letter inside a token.
```
Comment (vmarkovtsev) [inline lookout/style/typos/corruption.py]:
```suggestion
    Add a random letter inside a token.
```
(and all the places below)

File: lookout/style/typos/corruption.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    :param add_typo_probability: Probability with which one more corruption happens to a \
+                                 corrupted token.
+    :return: New dataframe with added columns Columns.CorrectToken and Columns.CorrectSplit, \
+             which contain tokens and corresponding splits from the `data`. Columns.Token and \
+             Columns.Split now contain partially corrupted tokens and corresponding splits.
+    """
+    result = data.copy()
+    result.loc[:, Columns.CorrectToken] = data[Columns.Token]
+    result.loc[:, Columns.CorrectSplit] = data[Columns.Split]
+    for i in tqdm(range(len(data))):
```
Comment (vmarkovtsev) [inline lookout/style/typos/corruption.py]: As I noted in the very first review, we should pay attention to the log level. https://github.com/src-d/style-analyzer/blob/master/lookout/style/format/rules.py#L883

File: lookout/style/typos/corruption.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+def rand_typo(token: str) -> str:
+    """
+    Make random typo in a token.
+    """
+    return random.choice([rand_insert, rand_delete, rand_substitution, rand_swap])(token)
+
+
+def corrupt_tokens_in_df(data: pandas.DataFrame, typo_probability: float,
+                         add_typo_probability: float) -> pandas.DataFrame:
+    """
```
Comment (vmarkovtsev) [inline lookout/style/typos/corruption.py]: https://github.com/src-d/style-analyzer/pull/603/files#diff-fc859ccee079f644b639fae5e0334d04R78

File: lookout/style/typos/corruption.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+def rand_typo(token: str) -> str:
+    """
+    Make a random typo in the token.
+    """
+    return random.choice([rand_insert, rand_delete, rand_substitution, rand_swap])(token)
+
+
+def corrupt_tokens_in_df(data: pandas.DataFrame, typo_probability: float,
+                         add_typo_probability: float, log_level: logging.Logger = logging.DEBUG,
```
Comment (vmarkovtsev) [inline lookout/style/typos/corruption.py]:
```suggestion
                         add_typo_probability: float, log_level: int = logging.DEBUG,
```

**Comment by irinakhismatullina** [comment]
Did that already

**Pull request merged**